src/libraries/sendtohelixhelp.proj

```diff
@@ -237,14 +248,22 @@
       </HelixPostCommands>
     </PropertyGroup>
 
+    <ItemGroup Condition="'$(Scenario)' == 'BuildWasmApps'">
```

Comment:
I'm not following what this should do. How does the item group `HelixCorrelationPayload` get consumed?

Reply:
This gets used by the helix sdk. Eg. for 

` <HelixCorrelationPayload Include="$(EmSdkDir)"                             Destination="build/emsdk" />

.. it makes the contents of `$(EmSdkDir)` available under `build/emsdk` on helix, in the correlation payload directory.
And that we pass as `WasmBuildSupportDir=${HELIX_CORRELATION_PAYLOAD}/build` to the tests.

Reply:
Should be in the helix sdk in arcade.

Reply:
https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Helix/Sdk/tools/Microsoft.DotNet.Helix.Sdk.MonoQueue.targets#L56